parallel.go

```diff
@@ -101,6 +101,8 @@ func (p *Parallel) secure(pipe *Pipeline) {
 			}
 			if p.exception != nil {
 				p.exception.OnExcept(err)
+			}else {
+				panic(err)
```

Comment:
Currently, only the above error types will raise panic, other types of errors will be ignored by default, but you can call parallel.Except() method to add an exception handler that can handle the panic or just re-panic.